static/js/settings_org.js

```diff
@@ -10,86 +10,6 @@ exports.reset = function () {
     meta.loaded = false;
 };
 
-var org_profile = {
-    name: {
-        type: 'text',
-    },
-    description: {
-        type: 'text',
-    },
```

Comment:
Why does this not require adding a widget type to the `description` textarea?  I guess the answer is it does, since I tested and reloading the page after saving a description change shows we're not actually sending the descriptions to the server.

A nice way to verify the code in this change is to look at each change you have here, and line them up against the changes to the templates.  If you removed an item here that isn't a bool without a corresponding change to the template, that's probably a bug.  

(Also, you can test by after clicking "save changes", reloading the page and seeing if your changes go through.  And you should test each and every widget this way, including the "select" dropdowns).  Or alternatively look at the payloads in the "network" tab to confirm the right changes got sent.

As a more high-level note, whenever you're doing a refactor, it's important to brainstorm at least one approach like this that you can use to be 100% confident that your changes didn't accidentally break the feature through this kind of omission -- most bugs end up looking kinda like this.

Reply:
Ahh, I possibly missed that part while testing features. Yeah, I'll be careful while testing so that it doesn't end up creating a bug. And will do like to go through the "network" tab to confirm my changes, Thanks for letting me know this.